### PR TITLE
EVG-5843: Add sort to tasks for build aggregation

### PR DESCRIPTION
**Pull request by ybrill**
Pagination is done by taking the next _limit_ number of tasks greater than or equal to the key provided and giving a key for the next page as the _limit+1_ task. If the tasks are not sorted the _limit+1_ task will likely not be the correct key.

Deploying this will require replacing the build_id index on the database with a composite index of build_id and _id so the match/sort stages can be done efficiently.

### CODE REVIEW

File: model/task/db.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/evergreen-ci/evergreen/util"
 	"github.com/mongodb/anser/bsonutil"
 	"github.com/pkg/errors"
-	"gopkg.in/mgo.v2"
+	mgo "gopkg.in/mgo.v2"
```
Comment (bsamek) [inline model/task/db.go]: I don't think this is necessary, but honestly have never thought about this before. Was this done by your editor or by you?

Comment (ybrill) [reply model/task/db.go]: the editor.

Comment (john-m-liu) [reply model/task/db.go]: I think goimport does this because having the package name verbatim would not compile

Comment (bsamek) [reply model/task/db.go]: This did compile though, and my goimport hook doesn't do this. I think it's fine to merge either way though.

**Comment by ybrill** [comment]
evergreen merge

**Pull request merged**